docs/index.rst

```diff
@@ -6,54 +6,76 @@ provides a simple interface for performing aperture photometry of either a singl
 image or a directory with multiple images. It is designed to be easy to use for both
 non-programmers and programmers.
 
-
-Getting Started
-===============
-
 Installation
-------------
+============
+
 
 Testing
-+++++++
+-------
 
 If you are testing a pre-release version of stellarphot we recommend setting up
 a virtual environment and installing stellarphot in this environment.
 
-Creating an environment with `conda` or `mamba`:
+Only use one of the methods below for making a virtual environment.
 
-```bash
-mamba create -n stellarphot-test python=3.11
-mamba activate stellarphot-test
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `conda` or `mamba`::
 
-Creating an environment with `virtualenv`:
+    mamba create -n stellarphot-test python=3.11
+    mamba activate stellarphot-test
+    pip install git+https://github.com/mwcraig/stellarphot.git@update-docs
 
-```bash
-python3 -m venv stellarphot-test
-source stellarphot-test/bin/activate
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `virtualenv`::
+
+
+    python3 -m venv stellarphot-test
+    source stellarphot-test/bin/activate
+    pip install --pre --upgrade stellarphot
+
+
+Getting Started
+===============
 
 Overview
 --------
 
+You will go through this process to do photometry.
+
+#. You need to make some settings, like camera properties, observatory information, and passband maps. You may only need
+   to do this step once if you use the same equipment for all of your observations.
+#. Settings specific to an object need to be made:
```

Comment:
However, now the problem, when I open the second notebook `2 - Seeing profile and comparison stars.ipynb` it crashes running the cell with
```
cs = ComparisonAndSeeing()
```
because apparently it can't read the settings:
```
ERROR:root:21 validation errors for Camera
name
  Value error, name must not be empty or contain only whitespace. [type=value_error, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
gain.is-instance[UnitBase]
  Input should be an instance of UnitBase [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
gain.is-instance[Quantity]
  Input should be an instance of Quantity [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
gain.chain[str,function-plain[_validator()]]
  Value error, Cannot parse "" as a Quantity. It does not start with a number. [type=value_error, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
gain.chain[float,function-plain[_validator()]]
  Input should be a valid number, unable to parse string as a number [type=float_parsing, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/float_parsing
read_noise.is-instance[UnitBase]
  Input should be an instance of UnitBase [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
read_noise.is-instance[Quantity]
  Input should be an instance of Quantity [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
read_noise.chain[str,function-plain[_validator()]]
  Value error, Cannot parse "" as a Quantity. It does not start with a number. [type=value_error, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
read_noise.chain[float,function-plain[_validator()]]
  Input should be a valid number, unable to parse string as a number [type=float_parsing, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/float_parsing
dark_current.is-instance[UnitBase]
  Input should be an instance of UnitBase [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
dark_current.is-instance[Quantity]
  Input should be an instance of Quantity [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
dark_current.chain[str,function-plain[_validator()]]
  Value error, Cannot parse "" as a Quantity. It does not start with a number. [type=value_error, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
dark_current.chain[float,function-plain[_validator()]]
  Input should be a valid number, unable to parse string as a number [type=float_parsing, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/float_parsing
pixel_scale.is-instance[UnitBase]
  Input should be an instance of UnitBase [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
pixel_scale.is-instance[Quantity]
  Input should be an instance of Quantity [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
pixel_scale.chain[str,function-plain[_validator()]]
  Value error, Cannot parse "" as a Quantity. It does not start with a number. [type=value_error, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
pixel_scale.chain[float,function-plain[_validator()]]
  Input should be a valid number, unable to parse string as a number [type=float_parsing, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/float_parsing
max_data_value.is-instance[UnitBase]
  Input should be an instance of UnitBase [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
max_data_value.is-instance[Quantity]
  Input should be an instance of Quantity [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
max_data_value.chain[str,function-plain[_validator()]]
  Value error, Cannot parse "" as a Quantity. It does not start with a number. [type=value_error, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
max_data_value.chain[float,function-plain[_validator()]]
  Input should be a valid number, unable to parse string as a number [type=float_parsing, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/float_parsing
WARNING:ipyautoui.custom.filechooser:path given doesnt exist
```

I will note the `partial_photometry_settings.json` does appear to have been created by the first notebook and does appear to have all the settings that cleared from it.

This might be related to the issue you mentioned in your post about some notebooks not running right, I restarted the kernel for the second notebook and re-ran it and only got the final warning
```
WARNING:ipyautoui.custom.filechooser:path given doesnt exist
```

The second notebook (once it worked) did load a FITS image and rather nicely allowed setting the seeing parameters and comparison stars on the two panes.  

One issue is the labels (when I "Click to show labels") were WAY TOO LARGE and worthless, since the hundreds of labels overlapped.

![Screenshot with labels turned on](https://github.com/user-attachments/assets/345e8b2b-adf0-4169-a826-9483c50094e9)

Another  issue is the lack of visible instructions instructions for the user on the screen... might I suggest having those panes OPEN from the start OR place some text to indicate to newbie users that clicking on those triangles will pop open the instructions?  Also, esthetically, its weird that the instructions for 
```
Click and drag or use arrow keys to pan, use +/- keys to zoom
```
and
```
Shift-left click (or Crtl-left click)to exclude star as target or comp. Click again to include.
```
are different font sizes... (and you need a space before "to exclude" in the second instruction).

Reply:
As mentioned above I did get the `WARNING:ipyautoui.custom.filechooser:path given doesnt exist` warning but have learned to just ignore warnings but this may be something that new people do not have ingrained in them yet. 

One thing that I noticed but am not sure what we would like to do about it if anything is that unsurprisingly notebook 2 does not like colr images. It create a wierd grid structure which I am thinking is the Bayer pattern from the camera. The image will correctly load and stars will be found but it just looks strange. 
![image](https://github.com/user-attachments/assets/637e9532-3221-426d-bee3-813e2197cf45)

The image is from my camera which is a ZWO 294 color camera.

Reply:
> However, now the problem, when I open the second notebook 2 - Seeing profile and comparison stars.ipynb it crashes running the cell with

> [...]

> ERROR:root:21 validation errors for Camera name
>  Value error, name must not be empty or contain only whitespace. [type=value_error, input_value='', input_type=str]
>    For further information visit https://errors.pydantic.dev/2.8/v/value_error
gain.is-instance[UnitBase]
>  Input should be an instance of UnitBase [type=is_instance_of, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/is_instance_of
This is a bug in `jupyter_app_launcher`, which I've reported here: https://github.com/trungleduc/jupyter_app_launcher/issues/66#issuecomment-2253104543

In the meantime, the workaround is to close the notebook and reopen it be double-clicking on it in the file browser.

Reply:
> WARNING:ipyautoui.custom.filechooser:path given doesnt exist

This is also a bug, in `ipyautoui`, fixed here: https://github.com/maxfordham/ipyautoui/pull/323

Looks like release `0.7.19` includes this fix, so I'll see if this is foxed for that version and update our minimum requirement for ipyautoui.

Reply:
> One thing that I noticed but am not sure what we would like to do about it if anything is that unsurprisingly notebook 2 does not like colr images. 

I expect that-- at least for now color images are out of scope for the project.

Reply:
> Another issue is the lack of visible instructions instructions for the user on the screen... might I suggest having those panes OPEN from the start OR place some text to indicate to newbie users that clicking on those triangles will pop open the instructions?

I moved the help instructions and legend up to the top. I think users will guess that they have to click on those to see more since that is pretty standard browser behavior.

I opened #389 as a reminder to pursue this further.

Reply:
> Also, esthetically, its weird that the instructions...are different font sizes

Fixed

Reply:
> One issue is the labels (when I "Click to show labels") were WAY TOO LARGE and worthless, since the hundreds of labels overlapped.

I mage the labels a bit smaller but in a field with a bunch of comparisons it will be hard to completely avoid overlap. I opened #390 to track this in case we want to pursue it down the road.